src/main/resources/templates/poker.html

```diff
@@ -3,17 +3,16 @@
 <head>
     <meta charset="UTF-8"/>
     <meta http-equiv="refresh" content="5"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title data-th-text="'Tech Poker (' + ${stats.currentVotes} + '/' + ${stats.playerVotes.size()} + ')'">Tech Poker</title>
     <script src="../static/js/jquery-3.1.1.min.js" data-th-src="@{/js/jquery-3.1.1.min.js}"></script>
     <script src="../static/js/bootstrap.min.js" data-th-src="@{/js/bootstrap.min.js}"></script>
     <script src="../static/js/qrcode.min.js" data-th-src="@{/js/qrcode.min.js}"></script>
     <script src="../static/js/techpoker-qr.js" data-th-src="@{/js/techpoker-qr.js}"></script>
     <script src="../static/js/techpoker-master.js" data-th-src="@{/js/techpoker-master.js}"></script>
 
-    <link rel="stylesheet" href="../static/css/bootstrap.min.css" data-th-href="@{/css/bootstrap.min.css}"
-          data-th-if="${currentSitePreference.normal}"/>
-    <link rel="stylesheet" href="../static/css/bootstrap.min.css" data-th-href="@{/css/theme/cyborg/bootstrap.min.css}"
-          data-th-if="${currentSitePreference.mobile || currentSitePreference.tablet}"/>
+    <link rel="stylesheet" href="../static/css/theme/darkly/bootstrap.min.css"
```

Comment:
I still think that white is nicer on a desktop environment and dark works for the mobile screens better. Also, Simon really wanted a true black background for mobile so that the amoled screens could turn off those pixels.

With a dark background, the QR code on the website actually needs a white border for it to be recognizable.

Reply:
Our team likes the dark theme - at least I've only heard positive voices so far. I added a media query that sets the background to black for narrower displays. This should work for anything under 960px in width. I no longer wanted to install the outdated and no longer supported Spring Boot Mobile Dependency.